server/pbench/bin/unittests

```diff
@@ -30,16 +36,24 @@ export _PBENCH_SERVER_TEST=1
 
 # execution environment
 _testopt=$_testroot/opt/pbench-server
-export PATH=$_testopt/unittest-scripts:$_tconfigtoolsbin:$PATH
+
+# All the "real" scripts are found at $_tdir, the mock scripts are found in
+# $_testopt/unittest-scripts. $_tconfigtoolsbin is not defined - should it? It should presumably
+# be the place where getconf.py is, but that's assumed to be installed already
+# in /usr/bin.
+export PATH=$_testopt/unittest-scripts:$_testopt/bin:/usr/bin
 export CONFIG=$_testopt/config/pbench-server.cfg
+# used by pbench-report-status
+export IDXCONFIG=$_testopt/config/pbench-index.cfg
 export TMPDIR=$_testtmp
 export TZ="UTC"
 
 function _run {
     tname=$1
     shift
     echo "+++ Running $tname" >> $_testout
-    $_tdir/$tname $@ >> $_testout 2>&1
+    command -v $tname
```

Comment:
Is this a necessary line?  It seems to emit the command to be executed before the test.  Should this go to `$_testout` or should we just remove it?

Reply:
No, it was another debugging line that I forgot. I got rid of it, but I'm now looking at the sed failure. I'll update asap.